### PR TITLE
Create announcements week20.md

### PR DESCRIPTION
**Pull request by cokiengchiara**
- [x] Apps - @cokiengchiara 
- [x] Docs - @marians 
- [x] UIs - @marians

### CODE REVIEW

File: announcements/2021/week20.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# :zap: This week in provider-independent announcements (May 21, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+
```
Comment (ubergesundheit) [inline announcements/2021/week20.md:7]:
```suggestion
[Prometheus Operator v0.8.2](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.2/) upgrades prometheus to v2.26.1 which contains a fix for [CVE-2021-29622](https://github.com/prometheus/prometheus/security/advisories/GHSA-vx57-7f4q-fpc7). The vulnerability allowed attackers to redirect to arbitray URLs through a specially crafted address (e.g.: http://127.0.0.1:9090/new/new<url>).
```

File: announcements/2021/week20.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# :zap: This week in provider-independent announcements (May 21, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Cloudflared v0.1.0](https://docs.giantswarm.io/changes/managed-apps/cloudflared-app/v0.1.0/) is now available. The app allows you to launch Cloudflare Argo Tunnels that route directly to services inside your cluster. Useful for clusters not running in a public cloud provider.
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:7]: @yasn77 pls check :)

File: announcements/2021/week20.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+[Prometheus Operator v0.8.2](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.2/) upgrades prometheus to [v2.26.1](https://github.com/prometheus/prometheus/releases/tag/v2.26.1) which contains a fix for [CVE-2021-29622](https://github.com/prometheus/prometheus/security/advisories/GHSA-vx57-7f4q-fpc7). The vulnerability allowed attackers to redirect to arbitray URLs through a specially crafted address (e.g.: http://127.0.0.1:9090/new/new<url>). Prior to that, [v0.8.1](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.1/) eases installing on kubernetes >= 1.18 by enabling `ingressClass` and `pathType` by default.
+
+[EFK v0.5.2](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.2/) reverts upstream changes of deployment labels that caused upgrade conflicts.
+
+[Cert Manager v2.7.1](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.7.1/) sets authoritative nameservers to `coredns` when using `dns01` ACME solver, in order to support DNS01 challenge introduced in [v2.6.0](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.6.0/).
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:13]: @glitchcrab @webwurst pls check :)

File: announcements/2021/week20.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## User interfaces
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:15]: @axbarsan pls check :)

File: announcements/2021/week20.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+[kubectl gs v1.28.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.28.0/) adds `get appcatalogs` and `get apps` commands. It also now allows templating for China clusters.
+
+We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to 1.9.0 to improve the user experience, as well as prepare for new features. [CAPI? Management API?]
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:19]: @giantswarm/chapter-front-end please check :)

File: announcements/2021/week20.md (hunk 3 above)
Comment (glitchcrab) [reply announcements/2021/week20.md:13]: seems sensible :)

File: announcements/2021/week20.md (hunk 5 above)
Comment (cokiengchiara) [reply announcements/2021/week20.md:19]: is the 'preparing for new features' related to a bigger initiative like CAPI or MAPI?

File: announcements/2021/week20.md (hunk 3 above)
Comment (cokiengchiara) [reply announcements/2021/week20.md:13]: thanks!

File: announcements/2021/week20.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+[EFK v0.5.2](https://docs.giantswarm.io/changes/managed-apps/efk-stack-app/v0.5.2/) reverts upstream changes of deployment labels that caused upgrade conflicts.
+
+[Cert Manager v2.7.1](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.7.1/  ) sets authoritative nameservers to `coredns` when using `dns01` ACME solver, in order to support DNS01 challenge introduced in [v2.6.0](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.6.0/).
+
+## Apps supported with best effort
+
+[Spark Operator](https://github.com/giantswarm/spark-operator), an operator that makes specifying and running Spark applications as easy as running other workloads on Kubernetes, is now available in the playground catalog.
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:17]: @pipo02mix pls check!

**Review by pipo02mix — APPROVED** [review]
LGTM

File: announcements/2021/week20.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# :zap: This week in provider-independent announcements (May 21, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Cloudflared v0.1.0](https://docs.giantswarm.io/changes/managed-apps/cloudflared-app/v0.1.0/) is now available. The app allows you to launch Cloudflare Argo Tunnels that route directly to services inside your cluster. Useful for clusters not running in a public cloud provider.
```
Comment (yasn77) [inline announcements/2021/week20.md:7]:
```suggestion
[Cloudflared v0.1.0](https://docs.giantswarm.io/changes/managed-apps/cloudflared-app/v0.1.0/) is now available. The app allows you to launch Cloudflare Argo Tunnels that route directly to services inside your cluster. Useful for clusters not running in a public cloud provider. This release significantly eases the deployment in Kubernetes by creating a single deployment that reuses a named tunnel.
```

File: announcements/2021/week20.md (hunk 2 above)
Comment (yasn77) [reply announcements/2021/week20.md:7]: See my suggested update 👇🏽

**Review by yasn77 — CHANGES_REQUESTED** [review]

File: announcements/2021/week20.md (hunk 5 above)
Comment (axbarsan) [reply announcements/2021/week20.md:23]: Yes, we're going full speed ahead towards MAPI

File: announcements/2021/week20.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+## Apps supported with best effort
+
+[Spark Operator](https://github.com/giantswarm/spark-operator), an operator that makes specifying and running Spark applications as easy as running other workloads on Kubernetes, is now available in the playground catalog.
+
+## User interfaces
+
+[kubectl gs v1.28.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.28.0/) adds `get appcatalogs` and `get apps` commands. It also now allows templating for China clusters.
+
+We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to 1.9.0 to improve the user experience, as well as prepare for new features. [CAPI? Management API?]
```
Comment (axbarsan) [inline announcements/2021/week20.md:23]:
```suggestion
We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to v1.9.0 to improve the user experience, as well as prepare for new features. [CAPI? Management API?]
```

File: announcements/2021/week20.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+# :zap: This week in provider-independent announcements (May 21, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
```
Comment (puja108) [inline announcements/2021/week20.md:3]: sidenote: we recently realized that what we write in these announcements is much nicer than what we have in the changes and releases, cause those are just from the changelogs, nothing to do on this right now, but we should see if at some point we can better integrate the two

Comment (cokiengchiara) [reply announcements/2021/week20.md:3]: yes! that's what this issue https://github.com/giantswarm/giantswarm/issues/17071 is about right?

File: announcements/2021/week20.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+## Apps supported with best effort
+
+[Spark Operator](https://github.com/giantswarm/spark-operator), an operator that makes specifying and running Spark applications as easy as running other workloads on Kubernetes, is now available in the playground catalog.
+
+## User interfaces
+
+[kubectl gs v1.28.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.28.0/) adds `get appcatalogs` and `get apps` commands. It also now allows templating for China clusters.
+
+We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to 1.9.0 to improve the user experience, as well as prepare for new features. [CAPI? Management API?]
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:23]:
```suggestion
We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to v1.9.0 to improve the user experience, as well as prepare for new features related to providing access to the Management API.
```
```suggestion
We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to 1.9.0 to improve the user experience, as well as prepare for new features. [CAPI? Management API?]
```

File: announcements/2021/week20.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+## Apps supported with best effort
+
+[Spark Operator](https://github.com/giantswarm/spark-operator), an operator that makes specifying and running Spark applications as easy as running other workloads on Kubernetes, is now available in the playground catalog.
+
+## User interfaces
+
+[kubectl gs v1.28.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.28.0/) adds `get appcatalogs` and `get apps` commands. It also now allows templating for China clusters.
+
+We recently made many fixes and changes to [The Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to v1.9.0 to improve the user experience, as well as prepare for new features related to providing access to the Management API.
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:23]:
```suggestion
We recently made many fixes and changes to [the Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to v1.9.0 to improve the user experience, as well as prepare for new features related to providing access to the Management API.
```

File: announcements/2021/week20.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+
+## User interfaces
+
+[kubectl gs v1.28.0](https://docs.giantswarm.io/changes/kubectl-gs/kubectl-gs/v1.28.0/) adds `get appcatalogs` and `get apps` commands. It also now allows templating for China clusters.
+
+We recently made many fixes and changes to [the Web UI](https://docs.giantswarm.io/changes/web-ui/) from v1.7.2 to v1.9.0 to improve the user experience, as well as prepare for new features related to providing access to the Management API.
+
+## Documentation
+
+Many docs are updated and made more accurate including: [Advanced CoreDNS Configuration](https://docs.giantswarm.io/advanced/coredns/), [App CR's target namespace configuration](https://docs.giantswarm.io/app-platform/namespace-configuration/), and [Usage data recording in our web interface](https://docs.giantswarm.io/ui-api/web/usage-data/)
```
Comment (cokiengchiara) [inline announcements/2021/week20.md:27]:
```suggestion
Many docs are updated and made more accurate including: [Advanced CoreDNS Configuration](https://docs.giantswarm.io/advanced/coredns/), [App CR's target namespace configuration](https://docs.giantswarm.io/app-platform/namespace-configuration/), and [Usage data recording in our web interface](https://docs.giantswarm.io/ui-api/web/usage-data/).
```

File: announcements/2021/week20.md (hunk 2 above)
Comment (cokiengchiara) [reply announcements/2021/week20.md]: thank you!!!

**Pull request merged**